### DISCUSSION TITLE
EN Wikipedia page header

### DISCUSSION THREAD

**Discussion by LunarTwilight** [darkreader/darkreader#8215] [Q&A]
It seems on the English Wikipedia, Dark Reader makes the page header not match the rest of the page's background (as seen in https://github.com/darkreader/darkreader/issues/8160#issuecomment-1034338261), but doesn't do this on other wikis (eg WMF's meta wiki). Would this be a bug with Dark Reader or an issue with the wiki's CSS? And if it's an issue with the wiki's CSS is it something that should be fixed or just something that I should keep as a personal fix?

For anyone interested, I've included my personal fix below:
```css
#mw-page-base {
    background: unset;
}
```

**Comment by Gusted** [comment]
> Would this be a bug with Dark Reader or an issue with the wiki's CSS? And if it's an issue with the wiki's CSS is it something that should be fixed or just something that I should keep as a personal fix?

This is a common bug on sites with dark reader, the header is actually using the body's background-color which currently mismatch `.mw-body`'s background color.

**Reply by LunarTwilight** [reply]
Ah so should it be something fixed in dark reader's dev tools?

**Reply by Gusted** [reply]
Once we fix it for one website, everyone can justify to have fix for any kind of site that also experiences this. It's at this moment a technical limitation of dark reader, which should be rather fixed in the code.
